components/eventing-controller/controllers/subscription/testing/test_utils.go

```diff
@@ -182,17 +199,19 @@ func createSubscriberSvcInK8s(ens *TestEnsemble) {
 	if ens.SubscriberSvc.Namespace != "default " {
 		namespace := fixtureNamespace(ens.SubscriberSvc.Namespace)
 		if namespace.Name != "default" {
-			err := ens.K8sClient.Create(ens.Ctx, namespace)
-			if !k8serrors.IsAlreadyExists(err) {
-				fmt.Println(err)
-				g.Expect(err).ShouldNot(gomega.HaveOccurred())
-			}
+			g.Eventually(func() error {
+				if err := ens.K8sClient.Create(ens.Ctx, namespace); !k8serrors.IsAlreadyExists(err) {
+					return err
+				}
+				return nil
+			}, SmallTimeout, SmallPollingInterval).ShouldNot(gomega.HaveOccurred(), "unable to create the namespace for the subscriber")
```

Comment:
This is how it is rendered by gomega. I decided to go with the upper case version because `Expected` also uses upper case :)
<img width="359" alt="Screenshot 2022-08-17 at 17 10 11" src="https://user-images.githubusercontent.com/8013145/185175877-60913486-2fb7-48a4-962c-cd171bd07ea5.png">